lab_1_classify_by_unigrams/start.py

```diff
@@ -14,7 +14,20 @@ def main() -> None:
         de_text = file_to_read_de.read()
     with open("assets/texts/unknown.txt", "r", encoding="utf-8") as file_to_read_unk:
         unknown_text = file_to_read_unk.read()
-    result = None
+    result = func.tokenize(en_text)
+    print(result)
+    print(func.calculate_frequencies(result))
+    print(func.create_language_profile('en', en_text))
+    en_profile = func.create_language_profile('en', en_text)
+    du_profile = func.create_language_profile('de', de_text)
+    un_profile = func.create_language_profile('un', unknown_text)
+    if en_profile is None or du_profile is None or un_profile is None:
+        return
+    detection_result = func.detect_language(un_profile, du_profile, en_profile)
```

Comment:
assign the output from this function to `result` variable

Reply:
done